packs/email/sensors/imap_sensor.py

```diff
@@ -35,13 +47,17 @@ def setup(self):
     def poll(self):
         self._logger.debug('[IMAPSensor]: entering poll')
 
-        for name, mailbox in self._mailboxes.items():
-            self._poll_for_unread_messages(name, mailbox)
+        for name, values in self._mailboxes.items():
```

Comment:
We don't consistently use `six` in st2contrib, we should probably consider adding `six` to the base requirements list which are installed with every pack.